python/GangaLHCb/Lib/LHCbDataset/LHCbDatasetUtils.py

```diff
@@ -38,13 +38,13 @@ def strToDataFile(name, allowNone=True):
         else:
             raise GangaException( "Cannot construct file object: %s" % str(name) )
 
-def getDataFile(file):
-    if isType(file, DiracFile):
-        return file
-    if isType(file, LocalFile):
-        return file
-    if type(file) == type(''):
-        return strToDataFile(file)
+def getDataFile(_file):
```

Comment:
PEP 8 suggests `single_trailing_underscore_` is "used by convention to avoid conflicts with Python keyword".